stage7/01-adi-kernel-dts/00-run.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 set -e
+
 SCRIPTS_DIR=wiki-scripts
 LINUX="linux-adi"
 
+declare -a klist=("zynq" "zynqmp" "socfpga")
 declare -a rpi_configs=("rpi" "2709" "2711")
 declare -a rpi_knames=("" "7" "7l")
 
 build_rpi_linux() {
 
-	# this will checkout the branch that has newest commit - should use a parameter/variable to be able to switch between branches
 	git checkout $(git branch -a --sort=-committerdate | grep -E 'rpi\-[0-9]{1,2}\.[0-9]{1,2}\.y$' | head -1 | sed -e 's/^remotes\/origin\///' -e 's/* //')
```

Comment:
At some point we'll need to add this under a variable or find another method to be able to select the branch. 
For example, if we want now to build 2019_R2 release, it will automatically checkout rpi-5.10.y branch that not the one for 2019_R2 release.

Reply:
That will be in a separate commit